docs/design/Ratify Vulnerability Report Verifier.md

```diff
@@ -21,7 +21,7 @@ Vulnerability reports are JSON documents attached to the subject as a referrer.
 
 ## SARIF Format
 
-SARIF documents provide a schema framework for static analysis reporting. In a nut shell, a set of rules configured on a tool's driver are applied to a subject. The results of each rule are outputted in a list with details on violations. There are many optional and custom fields that can be added to rules and results. Official specification [here](https://docs.oasis-open.org/sarif/sarif/v2.0/sarif-v2.0.html)
+SARIF documents provide a schema framework for static analysis reporting. In a nut shell, a set of rules configured on a tool's driver are applied to a subject. The results of each rule are outputted in a list with details on violations. There are many optional and custom fields that can be added to rules and results. Official specifications can be found [here](https://groups.oasis-open.org/communities/tc-community-home2?CommunityKey=c64ae352-bebf-446d-8ebf-018dc7d3eeb0#:~:text=Overview.%20SARIF%20TC%20members%20are%20developing)
```

Comment:
The previous link seems to be fine. I can access it with no issues. There is a newer version for it though: https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html

The link that you provided in this PR doesn't seem to be the actual SARIF specifications, it's SARIF TC.

Reply:
looks like they just fixed it :). I'll close this PR then